src/libwalletqt/Wallet.cpp

```diff
@@ -87,9 +88,20 @@ Wallet::Status Wallet::status() const
     return static_cast<Status>(m_walletImpl->status());
 }
 
-Wallet::ConnectionStatus Wallet::connected() const
+Wallet::ConnectionStatus Wallet::connected()
```

Comment:
It seems weird. Method supposed to be accessor but it actually changes state of the object.